src/main/java/org/folio/inventory/resources/Instances.java

```diff
@@ -603,7 +694,7 @@ private CompletableFuture<Instance> fetchPrecedingSucceedingTitles(
   // Utilities
 
   private CqlQuery cqlMatchAnyByInstanceIds(List<String> instanceIds) {
-    return CqlQuery.exactMatchAny("instanceId", instanceIds);
+    return CqlQuery.exactMatchAny(RelatedInstance.INSTANCE_ID_KEY, instanceIds);
```

Comment:
Whilst I understand the value is the same, I think that is by coincidence and the reuse of the constant could be confusing for folks, especially given that these concepts are already easy to mix up (given their similar names and structure)

(same for preceding / succeeding titles)

Reply:
Would you like I switch back to using string literals? There is one class in which causes a code smell due to the duplication of string literals.

Reply:
> Would you like I switch back to using string literals? There is one class in which causes a code smell due to the duplication of string literals.

I think this is an issue with static analysis.

I think it's ok to use a constant. I would prefer the constant was localised to that class and name more generally than specifically for related instances.